### PR TITLE
Handle no lyrics found errors

### PR DESCRIPTION
**Pull request by briana-mcn**
I noticed if no lyrics were found from the Metrolyrics API the errors weren't handled gracefully. Here is my attempt at handling those

### CODE REVIEW

File: exceptions.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class TswiftError(Exception):
+    """Base exception for errors raised in tswift"""
```
Comment (brenns10) [inline exceptions.py]: This should not be a separate file, for a couple reasons:
1. The `setup.py` only [declares](https://github.com/brenns10/tswift/blob/master/setup.py#L15) that one module (tswift) is part of this package. So when this package gets uploaded to PyPI, the `exceptions.py` file will not be included.
2. Even if we updated the `setup.py` to include the `exceptions` module, we'd encounter another problem. `exceptions` is a common name for Python modules, and as such we would be overwriting a namespace that many people would be using. A better module to put it in would be `tswift.exceptions`, but that would require reorganizing more code.

So please put this definition into `tswift.py`

File: tswift.py [hunk 2]
```diff
@@ -61,13 +63,22 @@ def __init__(self, title=None, artist=None, url=None):
...
-        verses = [c.text_content() for c in lyric_div.find_class('verse')]
-        self._lyrics = '\n\n'.join(verses)
+        try:
+            lyric_div = tree.get_element_by_id('lyrics-body-text')
+            verses = [c.text_content() for c in lyric_div.find_class('verse')]
+        except KeyError:
+            raise TswiftError("No lyrics available for requested song")
+        else:
+            self._lyrics = '\n\n'.join(verses)
+
```
Comment (brenns10) [inline tswift.py:84]: I'm able to reproduce the "KeyError" that happens when you load a song which doesn't exist, e.g. `tswift 'Taylor Swift' -s 'Not a song'`. But can you provide an example which triggers a 200 response with no `lyrics-body-text` element?

**Review by brenns10 — CHANGES_REQUESTED** [review]
Thanks for the CR! We definitely don't have any consideration for a "song not found" exception so this is a good idea. Looking forward to see a second revision.

File: tswift.py (hunk 2 above)
Comment (briana-mcn) [reply tswift.py:84]: I was only able to to surface that specific error with a 200 response without the `lyrics-body-text` element. The other one is a non-200 response which I got if you spelled the artist name incorrectly or the artist doesn't exist. This one had no `lyrics-body-text` element either.

File: exceptions.py (hunk 1 above)
Comment (briana-mcn) [reply exceptions.py]: Thanks for the explanation. I pushed an update moving the exception into the `twsift.py` file

File: tswift.py (hunk 2 above)
Comment (brenns10) [reply tswift.py:84]: I've been browsing around Metrolyrics but haven't yet found a situation where you would have a 200 response on a valid song lyric link, but you wouldn't have a `lyrics-body-text` element. Was there a particular instance you encountered this problem with?

(In any case, I'm ok with including this change as-is, it seems possible that you could get this error through the `Song(url='...')` constructor, or some other way).

**Review by brenns10 — APPROVED** [review]

**Comment by brenns10** [comment]
This is released on PyPI now: https://pypi.org/project/tswift/0.7.0/

**Comment by brenns10** [comment]
Thanks for contributing :)

File: tswift.py (hunk 2 above)
Comment (briana-mcn) [reply tswift.py:84]: Yea, if you attempt to look up a song by an artist that does not exist or is misspelled
https://www.metrolyrics.com/sdfsdf-lyrics-talking-heads.html

**Comment by briana-mcn** [comment]
Cool! Thanks @brenns10 !

**Comment by briana-mcn** [comment]
And thank you for the thorough review!

**Pull request merged**